sklearn/decomposition/kernel_pca.py

```diff
@@ -259,7 +262,7 @@ def inverse_transform(self, X):
         "Learning to Find Pre-Images", G BakIr et al, 2004.
         """
         if not self.fit_inverse_transform:
-            raise ValueError("Inverse transform was not fitted!")
+            raise NotFittedError("Inverse transform was not fitted.")
```

Comment:
Well, fit was called but the ``fit_inverse_transform`` option was not specified. I am not entirely sure. @ogrisel any opinion?